### PR TITLE
make grid/row/col use flex

### PR DESCRIPTION
**Pull request by AmeeMorris**
### 💬 Description
Goodbye floats, goodbye minus margins, goodbye clearfixes... hello flex!

Mainly styling changes here, its a bit hard to parse so you might be better off grabbing this branch locally and looking at the file in your ide/checking out the new grid story

### CODE REVIEW

**Comment by AmeeMorris** [comment]
[Lets make Row / Col / Grid use flex](https://trello.com/c/iTzw6Rwf/2230-lets-make-row-col-grid-use-flex)

**Review by DavidHollins6 — APPROVED** [review]
love the amount of deletions here
